### ISSUE TITLE
Insecure! Logging of all keystrokes to a log file

### ISSUE THREAD

**Issue by im1x** [iXab3r/MicSwitch#38]
This is absolutely unacceptable. The program saves all keystrokes to a log file. This completely undermines the safety of users.
It is necessary to disable logging of keystrokes to the file.

Example of a log file:
```
2021-04-19 21:33:15,396 [HotkeyTracker] DEBUG Hotkey pressed: T, key: T, mouse: , wheel: None, modifiers: None [PoeShared.UI.Hotkeys.HotkeyTracker]
2021-04-19 21:33:16,036 [HotkeyTracker] DEBUG Hotkey pressed: E, key: E, mouse: , wheel: None, modifiers: None [PoeShared.UI.Hotkeys.HotkeyTracker]
2021-04-19 21:33:16,516 [HotkeyTracker] DEBUG Hotkey pressed: S, key: S, mouse: , wheel: None, modifiers: None [PoeShared.UI.Hotkeys.HotkeyTracker]
2021-04-19 21:33:17,140 [HotkeyTracker] DEBUG Hotkey pressed: T, key: T, mouse: , wheel: None, modifiers: None [PoeShared.UI.Hotkeys.HotkeyTracker]
```

**Comment by MehItsOk** [comment]
well that's a bit concerning. Where is the log file being stored.

**Comment by im1x** [comment]
> Where is the log file being stored.

In Win10, the log files are located here - C:\Users\<USER>\AppData\Roaming\MicSwitch\logs
You can click the button in the application to go to the log files folder 
![MicSwitch_fPUDQ3YXrm](https://user-images.githubusercontent.com/1444150/115783058-13bd7400-a3c5-11eb-82e2-a10d26c7e4f4.jpg)

**Comment by dignite** [comment]
Seems to log hotkey presses, for me it is the mouse button that I use to mute and unmute. Does not log ALL keystrokes, right?

**Comment by im1x** [comment]
> Seems to log hotkey presses, for me it is the mouse button that I use to mute and unmute. Does not log ALL keystrokes, right?

All keys that were pressed are saved to a file

**Comment by iXab3r** [comment]
Fixed in 1.0.184
Should be fixed now, these log strings will now be written only if debugMode flag is passed (-d) 

My apologies, logging configuration accidentally ended up in release version :-\
